### ISSUE TITLE
Don't have money on wallet when result status payIn is 'CREATED'

### ISSUE THREAD

**Issue by duykhanh91** [Mangopay/mangopay2-php-sdk#364]
When i call api PayIn sometime result return is have  'CREATED' status and don't have money on wallet.
Can you explain to me what is the difference between “created” and “succeed” status?
I was check on document but don't see any explain about this
thanks
![image](https://user-images.githubusercontent.com/39787610/68374511-9b82d480-0177-11ea-8544-0de8c3bc1071.png)

**Comment by LeyArthur** [comment]
Hi @duykhanh91, 

There is various way of having a "CREATED" status on a Payin, for exemple when you create a bankwire PayIn, this one take 2/3 days to arrive on our bank account and thereby will stay in "CREATED" status until we have the money. As soon as we have it, it will pass in "SUCCEEDED" status. 
An other exemple could be when you create a web payin, the status will be "created" until you didn't proceed the payment on the website". 
To sum up, most of the time the payin status is "CREATED" in a first place when the 3DS/AVS/PayIn on Payline side are being check etc .. 

Hope i help you, 
Regards, 

Arthur
